app/assets/stylesheets/application.css.scss.erb

```diff
@@ -27,6 +27,10 @@ body { padding-top: 60px; }
   margin-bottom: 100px;
 }
 
+table {
```

Comment:
Do we have tables outside of Agent descriptions that you're trying to style, or am I misunderstanding you?